### DISCUSSION TITLE
What is required for this to work in docker

### DISCUSSION THREAD

**Discussion by MichaelEvanchik** [QuestPDF/QuestPDF#319] [General]
i am assuming

FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
`RUN apt-get install libgdiplus -yq`

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Since this discussion was opened, QuestPDF now ships with all native dependencies bundled — no additional system packages are required in Docker environments.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
